cpp/oneapi/dal/io/detail/load_graph.hpp

```diff
@@ -180,6 +180,18 @@ struct fill_unfiltered_neighs<Graph, /* IsDirected = */ false> {
             unfiltered_neighs[++rows_vec_atomic[edges[u].second] - 1] = edges[u].first;
         });
     }
+
+    template <typename Vertex, typename Weight, typename AtomicEdge>
+    auto operator()(const weighted_edge_list<Vertex, Weight> &edges,
```

Comment:
What is it?

Reply:
It was added to support read of undirected weighted graphs